src/components/About.jsx

```diff
@@ -0,0 +1,91 @@
+import {
+	Drawer,
+	DrawerBody,
+	DrawerHeader,
+	DrawerFooter,
+	DrawerOverlay,
+	DrawerContent,
+	DrawerCloseButton,
+	Flex,
+	Heading,
+	Text,
+	Icon,
+	HStack,
+} from '@chakra-ui/react';
+import { Link } from 'react-router-dom';
+import { Github, Linkedin } from 'feather-icons-react';
+
+function Bio({ name, github, linkedin }) {
+	return (
+		<HStack>
+			<Text>{name}</Text>
+			<Link to={github}>
+				<Icon as={Github} />
+			</Link>
+			<Link to={linkedin}>
+				<Icon as={Linkedin} />
+			</Link>
```

Comment:
Per the AC, I was originally expecting photos for each of us. Was there a reason why that was scrapped?
![image](https://github.com/the-collab-lab/tcl-55-smart-shopping-list/assets/99099243/797c9453-6a26-49e8-8ca7-5f52e1f46ce4)

Reply:
We also left out photos for the same clutter reason as the button. If you add more to the about drawer you have to scroll. Right now everything fits without having to scroll even on really small devices. We can add them if we really want them but we also all have pictures on our LinkedIns.